docs/reference/3-admin-api/whats-new.md

```diff
@@ -3,7 +3,27 @@
 This section provides an overview of what's new in the latest versions of the
 Admin API.
 
-## Updates in Admin API v2.2 (Latest Release)
+## Updates in Admin API v2.3 (Latest Release)
+
+### Features
+
+* Tenant endpoints on V2: new read-only and management endpoints.
+* Support for multiple credentials per application.
+* Health check endpoints for Security and Admin databases.
```

Comment:
Are these separate endpoints? Or is it just that the Security and Admin database are included as part of the health check? Can you list the health check endpoint(s) here?